Emby.IsoMounting/IsoMounter/LinuxIsoManager.cs

```diff
@@ -99,18 +102,23 @@ public LinuxIsoManager(ILogger logger, IFileSystem fileSystem, IEnvironmentInfo
 
         #region Interface Implementation for IIsoMounter
 
-        public bool IsInstalled {
-            get {
+        public bool IsInstalled
+        {
+            get
+            {
                 return true;
             }
         }
 
-        public string Name {
+        public string Name
```

Comment:
I've seen these changed to `public string Name => "LinuxMount";` etc

Reply:
Yes, but that is a separate formatter, will run it after editorconfig-core is done.